frame/benchmarking/README.md

```diff
@@ -150,9 +150,13 @@ feature flag:
 
 ```bash
 cd bin/node/cli
-cargo build --release --features runtime-benchmarks
+cargo build --profile=production --features runtime-benchmarks
 ```
 
+The production profile applies various compiler optimizations.  
+These optimizations slow down the compilation process *a lot*.  
+If you are just testing things out and don't need final numbers, use `--release` instead.
```

Comment:
just don't use `--release` at all?

Reply:
Yea that is the TLDR. You think I should just remove it?  
My thought was that if you modify the source it would be horrible to wait for a production build every time.

Reply:
```suggestion
If you are just testing things out and don't need final numbers, don't include `--profile=production`.
```